modules/installation-and-upgrade/pages/hardware-requirements.adoc

```diff
@@ -20,7 +20,7 @@ For {productname} {smr} hardware requirements, see xref:retail:retail-requiremen
 [[server-hardware-requirements]]
 == Server Hardware Requirements
 
-{productname} Server stores packages in the [path]``/var/spacewalk/`` directory.
+By default the {productname} Server container stores packages in the [path]``/var/spacewalk/`` directory.
```

Comment:
This is the path inside the container, you probably want to highlight the path on the host and the `--podman-mount-[cache|postgresql|spacewalk]` options to override it.

Reply:
Added your comment to my notes for storage docs.